pycbc/workflow/pegasus_files/local-site-template.xml

```diff
@@ -1,10 +1,10 @@
   <site handle="local" arch="x86_64" os="LINUX">
-    <directory  path="$PWD" type="shared-scratch" free-size="null" total-size="null">
-        <file-server  operation="all" url="file://$PWD">
+    <directory  path="$LOCAL_SITE_PATH/local-site-scratch" type="shared-scratch" free-size="null" total-size="null">
+        <file-server  operation="all" url="$LOCAL_SITE_URL/local-site-scratch">
```

Comment:
Is there a reason you are changing the local scratch directory? I think people are just getting used to the current layout. If possible, I don't think we should change it.